utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -195,15 +195,14 @@ def install_toolchain_path(self, host_target):
         """toolchain_path() -> string
 
         Returns the path to the toolchain that is being created as part of this
-        build, or to a native prebuilt toolchain that was passed in.
+        build
         """
-        if self.args.native_swift_tools_path is not None:
-            return os.path.split(self.args.native_swift_tools_path)[0]
-
         install_destdir = self.args.install_destdir
         if self.args.cross_compile_hosts:
-            build_root = os.path.dirname(self.build_dir)
-            install_destdir = '%s/intermediate-install/%s' % (build_root, host_target)
+            if self.is_darwin_host(host_target):
+                install_destdir = self.get_host_install_destdir(host_target)
+            else:
+                install_destdir = os.path.join(install_destdir, self.args.host_target)
```

Comment:
I could only replace the Darwin line above with `get_host_install_destdir()` and not this line, because this line has to point at the separate host toolchain for non-Darwin, whereas Darwin places all targets together?

Reply:
Darwin places all targets together and then `lipo`s them into universal binaries.